gradle/libs.versions.toml

```diff
@@ -14,7 +14,7 @@ autoValue = "1.10"
 dagger = "2.39.1"
 espresso = "3.5.1"
 gradlePublish = "0.15.0"
-kotlin = "1.8.0"
+kotlin = "1.8.20"
```

Comment:
Can you also update the ext property `kotlinPreview`? It's currently set based on `1.8.0`

Reply:
[e4a604a](https://github.com/square/anvil/pull/689/commits/e4a604a8ef3e19bdfa8ef91a06f8d043263910e8)